authors/tests/test_report_article.py

```diff
@@ -0,0 +1,109 @@
+import jwt
+import json
+from django.conf import settings
+from rest_framework.views import status
+from django.urls import reverse
+from ..apps.articles.models import User, Article
+# local imports
+from .base_test import TestBase
+
+
+class TestBookmarkArticle(TestBase):
+    def add_article(self):
```

Comment:
Most of the logic in this setup function is already implemented in the TestBase class. Please keep to the DRY principle and use the available methods.